src/panels/developer-tools/state/developer-tools-state.js

```diff
@@ -157,7 +159,20 @@ class HaPanelDevState extends EventsMixin(LocalizeMixin(PolymerElement)) {
                 title="[[localize('ui.panel.developer-tools.tabs.states.more_info')]]"
               >
               </paper-icon-button>
-              <a href="#" on-click="entitySelected">[[entity.entity_id]]</a>
+              <paper-icon-button
+                id="copyBtn-[[sanitizeId(entity.entity_id)]]"
```

Comment:
What is the use of this?

Reply:
It might be a remaining of a previous usage of the tooltip component (the dot was breaking the id referencing). I'll check tonight after work, I'll probably remove it.

Reply:
I removed all the sanitizeID stuff. It was related to a previous usage of a tooltip and ids with a dot were messing with the behaviour of the component.